scripts/biomedical/diseaseOntology/disease_ontology.tmcf

```diff
@@ -0,0 +1,23 @@
+Node: E:DiseaseOntology->E1
+typeOf: dcs:Disease
+dcid: C:DiseaseOntology->dcid
+name: C:DiseaseOntology->label
+parent: C:DiseaseOntology->subClassOf
+description: C:DiseaseOntology->IAO_0000115
+alternativeDiseaseOntologyID : C:DiseaseOntology->hasAlternativeId 
+diseaseSynonym: C:DiseaseOntology->hasExactSynonym
+internationalClassificationOfDiseaseID: C:DiseaseOntology->ICDO
+medicalSubjectHeadingDescriptorID: C:DiseaseOntology->MESH
```

Comment:
Can you please confirm that all of these IDs start with D. If not, then they aren't all MeSH Descriptors and we should switch to using the more general "medicalSubjectHeadingID" property here.